scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Denotation.scala

```diff
@@ -0,0 +1,110 @@
+package scala.meta
+package semantic
+
+import org.scalameta.adt._
+import org.scalameta.data._
+
+@data class Denotation(flags: Long, name: String, info: String) extends HasFlags {
```

Comment:
I believe that this a very bad misuse of the term. This should be called Definition instead.